.github/workflows/go.yml

```diff
@@ -61,6 +58,6 @@ jobs:
         uses: codecov/codecov-action@v3
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
-          fail_ci_if_error: false
+          fail_ci_if_error: true
```

Comment:
Should this be true or false? @dasnellings

Reply:
I believe we decided this should be false. We want to be notified if coverage is going down, but we don't want to outright fail the PR.